core/services/relay/evm/functions/logpoller_wrapper_test.go

```diff
@@ -0,0 +1,47 @@
+package functions_test
+
+import (
+	"encoding/hex"
+	"testing"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	evmclimocks "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client/mocks"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	lpmocks "github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller/mocks"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/functions/config"
+	"github.com/smartcontractkit/chainlink/v2/core/services/relay/evm/functions"
+)
+
+func TestLogPollerWrapper_BasicEmptyEvents(t *testing.T) {
```

Comment:
Time-permitting, I'd really like to see to more tests than just an empty event.  At the very least, I would like to see a test which mocking OracleRequest and OracleResponse events on both an active and proposed coordinator contract.

Reply:
Emphasis on "time-permitting" ;) Integration test covers that case (next PR) but I'll obviously add more units next week.